src/components/footer/Footer.component.tsx

```diff
@@ -0,0 +1,35 @@
+import fbLogo from '../../assets/icons/facebook.svg';
+import instLogo from '../../assets/icons/instagram.svg';
+import linkedLogo from '../../assets/icons/linkedin.svg';
+
+import styles from './footer.module.css';
+
+const fbLink = 'https://uk-ua.facebook.com/';
+const linkedLink = 'https://www.linkedin.com/in/%D0%B1%D0%BE%D0%B3%D0%B4%D0%B0%D0%BD-%D1%8F%D1%86%D0%B5%D0%BD%D1%82%D1%8E%D0%BA-841247270/';
+const instagramLink = 'https://www.instagram.com/aipiservis/';
+const mastersLink = 'https://www.mastersacademy.education/';
+
+export const FooterComponent = () => (
+    <footer className={styles.footer}>
+        <div className={`${styles.footer__row} ${styles.content}`}>
+            <article className={styles.footer__socials}>
+                <a href={fbLink} className={styles.footer__link}>
+                    <img src={fbLogo} alt="facebook" className={styles.footer__icon} />
+                </a>
+                <a href={linkedLink} className={styles.footer__link}>
+                    <img src={linkedLogo} alt="facebook" className={styles.footer__icon} />
+                </a>
+                <a href={instagramLink} className={styles.footer__link}>
+                    <img src={instLogo} alt="facebook" className={styles.footer__icon} />
+                </a>
```

Comment:
Create a separate component from it
it should accept all native `a` tag attributes
Hint:
use `ComponentType` build in React interface 

Considering the repeated use of `<a>` tags with identical functionality and styling throughout the application, it would be beneficial to encapsulate this pattern into a reusable component. 
This approach will centralize common logic and styles, facilitating easier updates and ensuring consistency across the application. Please create a helper component for the `<a>` tags to streamline code maintenance and improve readability.

```tsx
<Link href target ...etc></Link>
```
component should be able accept any props which native `a` tag can accept

Hint: Go to definiton files of a tag you can do it by `ctrl + click` or `cmd + click` on MacOs
https://mortenbarklund.com/blog/react-typescript-props-spread/
![Screenshot 2024-04-19 at 18 10 36](https://github.com/Allocene/fe-react-2024/assets/55930010/e9051de3-4049-4ac1-b776-ab02dbe7ffc2)